doc/examples/vuhAndroid/app/CMakeLists.txt

```diff
@@ -0,0 +1,89 @@
+# For more information about using CMake with Android Studio, read the
+# documentation: https://d.android.com/studio/projects/add-native-code.html
+
+# Sets the minimum version of CMake required to build the native library.
+
+cmake_minimum_required(VERSION 3.8)
+
+# for shaderc
+SET(SHADERC_SRC ${ANDROID_NDK}/sources/third_party/shaderc)
```

Comment:
it's an example for dynamic compile code, .spv file for better performance. 

for debug .comp file is better

it's just an example

Reply:
AOT is better for performance (you do not compile comp->spv on every app start) and requires much less code. If there is no Android-specific reason to do dynamic compilation I'd prefer to stay constistent with other examples and precompile the shader into spv. I wouldn't mind if cmake helper for that use shaderc instead of glslang, but at config/compile time not at runtime.

Reply:
On Android, sometimes we need dynamic generate code ,not only Vulkan code ,but also java code ,or javascript code , so we can't decide which code to use . 

So use  shaderc is better. android does not have glslang . 

this example is from google's  example .

compile comp->spv will be add later. most time ,we need this.

Reply:
following https://github.com/LunarG/VulkanSamples.git

Reply:
or  https://github.com/googlesamples/vulkan-basic-samples.git